packages/sdk-js/src/auto-wrapper.ts

```diff
@@ -266,6 +297,44 @@ gb.init({
     dataContext.noStreaming ||
     windowContext.backgroundSync === false
   ),
+  cacheSettings: windowContext.cacheSettings,
```

Comment:
if antiFlicker set, hide the screen before init(). Otherwise we're waiting for features to load